Example/UI Examples/MockCustomerContext.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  MockCustomerContext.swift
+//  UI Examples
+//
+//  Created by Ben Guo on 7/18/17.
+//  Copyright © 2017 Stripe. All rights reserved.
+//
+
+import Foundation
+import Stripe
+
+class MockCustomer: STPCustomer {
+    var mockSources: [STPSourceProtocol] = []
+    var mockDefaultSource: STPSourceProtocol? = nil
+    var mockShippingAddress: STPAddress?
+
+    override init() {
+        // Preload the mock customer with saved cards
+        let visa = [
+            "id": "preloaded_visa",
+            "exp_month": "10",
+            "exp_year": "2020",
+            "last4": "1881",
```

Comment:
These last4s are from our test cards for these brands: https://stripe.com/docs/testing#cards
I thought it'd be better to not use "(42424242)4242" and "(55555555)4444" since those numbers are the easiest to type if you're testing adding a card to `PaymentMethods`.